extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -233,7 +233,7 @@ private YoutubeParsingHelper() {
      * The three digits at the end can be random, but are required.
      * </p>
      */
-    private static final String CONSENT_COOKIE_VALUE = "PENDING+";
+    private static final String CONSENT_COOKIE_VALUE = "YES+";
```

Comment:
This is to address https://github.com/TeamNewPipe/NewPipeExtractor/issues/820

Reply:
Happy to apply these changes, although if we only use the `YES+` cookie in tests, would that mean the mix extractor won't work for users in the EU?

Btw, any clue how I can test this extractor in NewPipe app? 😅 
I've tried searching for playlists, but none of them seem to be youtube mixes.